liquid/templates/conf/emails/startupfair_invite.txt

```diff
@@ -1,10 +1,10 @@
 Dear {{ company.first_name }},
 
-My name is {{ user.get_full_name }} and I am part of Founders, a student entrepreneurship group at the University of Illinois at Urbana-Champaign. Together with the student chapter of the Association for Computing Machinery, we are hosting an on-campus startup career fair this fall as part of the Reflections | Projections Conference. I am contacting you to see if {{ company.company_name }} is interested in participating.
+My name is {{ user.get_full_name }} and I am Startup fair chair of the Reflections | Projections 2016 Conference. We’re excited to extend an invitation to {{company.company_name}} to attend this year’s annual Reflections | Projections startup fair. The startup fair will be held on Thursday, September 29th, 2016, and we would love to have {{company.company_name}} involved.
```

Comment:
Minor wording nit: might replace "I am Startup fair chair of" with "I'm helping run the Startup Fair at"